drop/modules/mae-pipeline/MAE/ASEReadCounter.sh

```diff
@@ -57,10 +56,19 @@ for chr in $chr_subset; do
     tail -n+2 >>$tmp
 done
 
-echo $mae_id
 cat $tmp | awk -v id="${mae_id}" \
   -F $'\t' 'BEGIN {OFS = FS} NR==1{print $0, "ID"} NR>1{print $0, id}' |
   bgzip >${output}
 rm ${tmp}
 
 zcat ${output} | head
+
+num_out=$(zcat "${output}" | wc -l )
+if [ "${num_out}" -lt 2 ]
+then
+  echo 'ERROR: No allele-specific counts'
```

Comment:
Please add a suggestion how to solve it. 
1. Checking the chromosomes (bam/vcf)
2. Check the reference -> having 1 reference/fasta file per BAM could also solve dome of the issues here
3. Removing the sample from MAE
4. ...